File: unit0/meow.c
#include <cs50.h>
#include <stdio.h>

void meow(int n);

int main(void)
{
    printf("Line 8\n");
    meow(3);
    printf("Line 10\n");
}

void meow(int n)
{
    printf("Line 15\n");
    for (int i = 0; i < n; i++)
    {
        printf("meow\n");
    }
    printf("Line 20\n");
}

File: unit3/meow.py
def meow(n):
    print("Line 2")
    for i in range(n):
        print("meow")
    print("Line 5")

def main():
    print("Line 8")
    meow(3)
    print("Line 10")

# print("Line 12")
# main()
# print("Line 14")

if __name__ == "__main__":
    main()